libs/core/coroutines/include/hpx/coroutines/detail/get_stack_pointer.hpp

```diff
@@ -28,28 +29,6 @@
 #include <limits>
 
 namespace hpx::threads::coroutines::detail {
```

Comment:
Since the function is defined only for `#if !defined(HPX_WINDOWS)`, shouldn't it be declared only in that case as well?

Reply:
I think what you are describing is what already happens since https://github.com/STEllAR-GROUP/hpx/blob/7624b2e9006423f693757489d3e254f5e9d500d1/libs/core/coroutines/include/hpx/coroutines/detail/get_stack_pointer.hpp#L15 extends to the bottom of the document